src/command-line/users/add.js

```diff
@@ -32,33 +32,48 @@ program
 		log.prompt({
 			text: "Enter password:",
 			silent: true,
-		}, function(err, password) {
+		}, (err, password) => {
 			if (!password) {
 				log.error("Password cannot be empty.");
 				return;
 			}
-			if (!err) {
+
+			if (err) {
+				return;
+			}
+
+			log.prompt({
+				text: "Save logs to disk?",
+				default: "yes",
+			}, (err2, enableLog) => {
+				if (err2) {
+					return;
+				}
+
 				log.prompt({
-					text: "Save logs to disk?",
-					default: "yes",
-				}, function(err2, enableLog) {
-					if (!err2) {
-						add(
-							manager,
-							name,
-							password,
-							enableLog.charAt(0).toLowerCase() === "y"
-						);
+					text: "Admin?",
+					default: "no",
+				}, (err3, isAdmin) => {
+					if (err3) {
+						return;
 					}
+
+					add(
+						manager,
+						name,
+						password,
+						isAdmin.charAt(0).toLowerCase() === "y",
+						enableLog.charAt(0).toLowerCase() === "y"
+					);
 				});
-			}
+			});
 		});
 	});
 
-function add(manager, name, password, enableLog) {
+function add(manager, name, password, isAdmin, enableLog) {
 	var hash = Helper.password.hash(password);
-	manager.addUser(name, hash, enableLog);
+	manager.addUser(name, hash, isAdmin ? "admin" : "", enableLog);
 
-	log.info(`User ${colors.bold(name)} created.`);
+	log.info(`${isAdmin ? "Admin" : "User"} ${colors.bold(name)} created.`);
```

Comment:
There are a few times we get this, could we just use the role from the user object?

Reply:
If we go for the `LocalUser` route, we can surely add helpers in there for that. But since that'd require upper-casing the first letter in some cases, and not in others, we'd have to do  `user.friendlyRole({capitalize: true})` or something silly like that.
If not, we can still do that, but that adds a silly helper to `client.js` so meh :man_shrugging: